src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './calculator.css';
+/* eslint-disable */
+class Calculator extends React.Component {
+  render() {
+    return (
+
+      <div className="calculator">
+        <input type="text" className="calculator-screen" value="0" disabled />
```

Comment:
- [ ] According to the project requirement your component should be a functional component. Please implement it. Example
````jsx
import React from "react"

function Calculator() {
  return <div>{/* Calculator app design goes here */}</div>
}

export default Calculator // export Calculator component
```